### DISCUSSION TITLE
[Support]: System logs no longer have a vertical scrollbar thus are unsearchable

### DISCUSSION THREAD

**Discussion by krojew** [blakeblackshear/frigate#13175] [General Support]
### Describe the problem you are having

In the old UI, logs had a normal vertical scrollbar so it was easy to look through them or jump to specific point. Now that scrollbar is gone which makes effectively impossible to find anything when the amount of entries is high.

### Steps to reproduce

Gather a lot of logs and go to the system logs page. Observe there's no way to jump to a speficic point or scroll quickly.

### Version

0.14.0-da913d8

### Frigate config file

```yaml
irrelevant to the issue
```

### Relevant log output

```shell
irrelevant to the issue
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Reolink

**Comment by hawkeye217** [comment]
This is not a bug and is by design. The design and functionality can certainly be improved. Additionally, there is a feature request to make the logs more relevant when messages are repeated: https://github.com/blakeblackshear/frigate/issues/12455

However, for bulk-viewing logs, we've always recommended using `docker logs`: https://docs.frigate.video/troubleshooting/faqs#how-can-i-view-the-frigate-log-files-without-using-the-web-ui
